tests/kernel/before_nfs_test.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Preparation before provisioning NFS setup
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+
+use Mojo::Base "opensusebasetest";
+use testapi;
+use serial_terminal "select_serial_terminal";
+use utils;
+use lockapi;
+
+sub run {
+    my ($self) = @_;
+
+    select_serial_terminal;
+    systemctl 'stop ' . $self->firewall;
+    set_hostname(get_var("HOSTNAME", "susetest"));
+}
```

Comment:
May need to wait a "grace time" here to wait for NFS v4 server get ready. e.g. sleep 2 * `cat /proc/fs/nfsd/nfsv4gracetime`.